Userland/Utilities/tar.cpp

```
@@ -197,23 +170,27 @@ int main(int argc, char** argv)
                 return;
             }
 
-            struct stat statbuf;
-            if (lstat(path.characters(), &statbuf) < 0) {
+            auto statbuf_or_error = Core::System::lstat(path.characters());
+            if (statbuf_or_error.is_error()) {
                 warnln("Failed stating {}", path);
                 return;
```

This should return the error as well.

---

Fixed.